runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -211,7 +211,13 @@ export function get_page_handler(
 			}
 
 			if (preload_error) {
-				handle_error(req, res, preload_error.statusCode, preload_error.message);
+				if (!error) {
+					handle_error(req, res, preload_error.statusCode, preload_error.message);
+				}
+				else {
```

Comment:
We'd normally put this all on one line like: `} else {`

Reply:
Fixed it. How about checking in a `.prettierrc` and running it all through prettier? At the moment I have to format by hand because running files through prettier makes too many changes.

Reply:
I've never used prettier, but am open to the idea of at least seeing what code formatted with prettier looks like. Up until this point I'd been avoiding it because I kept seeing so many issues reported in prettier-plugin-svelte, but perhaps you've fixed enough of those now :smile:

Reply:
I improved the linting rules instead to check for this in #1573